### DISCUSSION TITLE
Manual database migration

### DISCUSSION THREAD

**Discussion by bongani-chirwa** [planetscale/discussion#450] [Q&A] [answered]
Hi, I am trying to manually import a database with records but I keep receiving this error message "foreign key constraints are not allowed" any suggestions to overcome this?

**Comment by fooforge** [comment, marked as answer]
@bongani-chirwa PlanetScale does not support foreign key constraints yet. We have strategies how to operate without foreign key constraints in [our documentation](https://planetscale.com/docs/learn/operating-without-foreign-key-constraints).
